apps/server/src/modules/tool/external-tool/controller/tool.controller.ts

```diff
@@ -252,4 +256,25 @@ export class ToolController {
 
 		return streamableFile;
 	}
+
+	@Get('medium/:mediumId/media-source/:format/:mediaSourceId/metadata')
```

Comment:
This should be under something like `/medium-metadata/medium/:mediumId/media-source/:mediaSourceId/`
The format serves no purpose here since it can be identified by the MediaSource.